qemu/tests/hpt_miscellaneous.py

```diff
@@ -38,6 +41,18 @@ def verify_hpt(test, params, session, hpt_size):
                           % (get_hpt_value, hpt_size))
 
 
+def check_mem_increase(session, params, orig_mem, in_mem):
+    """Check the size of memory increased."""
+    new_mem = int(session.cmd_output(cmd=params['free_mem_cmd']))
+    if (new_mem - orig_mem) == in_mem:
```

Comment:
`if (new_mem - orig_mem) == in_mem:` -> `if (new_mem - orig_mem) == increase_mem:`

Reply:
Updated it from in_mem to increase_mem.